db/migrate/20230303214429_likes.rb

```diff
@@ -1,6 +1,6 @@
 class Likes < ActiveRecord::Migration[7.0]
   def change
-    create_table :likes, primary_key: [:user_id, :post_id] do |t|
+    create_table :likes, primary_key: %i[user_id post_id] do |t|
       t.references :user, null: false, foreign_key: true
       t.references :post, null: false, foreign_key: true
       t.timestamps
```

Comment:
- [ ] When I run `rails db:migrate`, I get an error that the relation likes already exists. Upon further investigation, I notice that you have 2 migration files for the likes model. I would advise you to delete this file so that no error pops up when running the migration file. 👍

Reply:
Sorry I deleted that file but I forgot to push

Reply:
@Mov305 it is all good. Great work once again 👍